dash/dash.py

```diff
@@ -440,7 +451,8 @@ def index(self, *args, **kwargs):  # pylint: disable=unused-argument
 
         index = self.interpolate_index(
             metas=metas, title=title, css=css, config=config,
-            scripts=scripts, app_entry=_app_entry, favicon=favicon)
+            scripts=scripts, app_entry=_app_entry, favicon=favicon,
+            renderer=renderer)
 
         checks = (
             (_re_index_entry_id.search(index), '#react-entry-point'),
```

Comment:
Can we add a check for the new renderer, it should check that `new DashRenderer` is somewhere in the renderer tag.